collect_app/src/main/java/org/odk/collect/android/javarosawrapper/FormController.java

```diff
@@ -884,13 +884,12 @@ public int jumpToIndex(FormIndex index) {
      */
     public void jumpToNewRepeatPrompt() {
         FormIndex repeatGroupIndex = getRepeatGroupIndex(getFormIndex(), getFormDef());
-        String repeatRef = repeatGroupIndex.getReference().toString(false);
-        String testRef = "";
+        Integer depth = repeatGroupIndex.getDepth();
```

Comment:
I like the use of depth here! I can really visualize how you go into a deeper part of the form within the repeat, maybe even deeper into nested repeats and then you reach the same depth at the new repeat event.